integration-cli/docker_api_inspect_test.go

```diff
@@ -6,9 +6,20 @@ import (
 	"net/http"
 	"strings"
 
+	"github.com/docker/docker/api/types"
 	"github.com/go-check/check"
 )
 
+func contains(tags []string, tag string) bool {
```

Comment:
A function like this exists here:
https://github.com/docker/docker/blob/master/pkg/stringutils/stringutils.go#L45

It's slightly different because it's is discarding case but I don't think that matters here. If so, let's use that.